dev/NavigationView/NavigationView.cpp

```diff
@@ -1387,10 +1387,15 @@ void NavigationView::RaiseItemInvoked(winrt::IInspectable const& item,
 // forceSetDisplayMode: On first call to SetDisplayMode, force setting to initial values
 void NavigationView::SetDisplayMode(const winrt::NavigationViewDisplayMode& displayMode, bool forceSetDisplayMode)
 {
-    if (forceSetDisplayMode || DisplayMode() != displayMode)
-    {
-        UpdateVisualStateForDisplayModeGroup(displayMode);
+    auto changingDisplayMode = DisplayMode() != displayMode;
```

Comment:
nit: move the declaration closer to usage? It's not used until the if statement, putting it above the if statement is easier to read (for me)